README.md

```diff
@@ -109,7 +114,79 @@ pnpm nx build dialtone-documentation
 ### Releasing
 
 ```bash
-pnpm run release:all
+pnpm run release:affected
 ```
 
 This will automatically release all packages that need to be released.
+
+## Installation
+
+### Install it via NPM:
+
+```shell
+npm install @dialpad/dialtone@next
+```
+
+### Import packages:
+
+#### Dialtone CSS
+
+- LESS
+
+```less
+@import "@dialpad/dialtone/css";
+```
+
+- Javascript
+
+```js
+import "@dialpad/dialtone/css";
+```
+
+#### Dialtone eslint-plugin
+
+```js
+import dialtone from "@dialpad/dialtone/eslint-plugin"
+```
+
+#### Dialtone icons
+
+Dialtone icons doesn't have a default export, so you need to access
+the files directly under the dist/ folder as following:
+
+- Importing icons:
+
+```js
+import IconArrowUp from '@dialpad/dialtone/dist/icons/svg/arrow-up.svg';
+```
+
+- Importing json files
+
+```js
+import keywords from '@dialpad/dialtone/dist/icons/keywords.json';
+import iconsList from '@dialpad/dialtone/dist/icons/icons.json';
+```
+
+#### Dialtone Vue
+
+- Vue 2
+
+```js
+import * from "@dialpad/dialtone/vue2"
+```
+
+- Vue 3
+
+```js
+import * from "@dialpad/dialtone/vue3"
+```
```

Comment:
Maybe lets do an example component instead of * since we never really want anyone importing *, and people tend to copy paste stuff directly from the docs.